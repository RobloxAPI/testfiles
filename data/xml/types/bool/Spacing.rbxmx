<roblox version="4">
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueBase</string>
			<bool name="Value">true</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueEmpty</string>
			<bool name="Value"></bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueEmptySpace</string>
			<bool name="Value">    </bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueEmptyTab</string>
			<bool name="Value">	</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueEmptyNewline</string>
			<bool name="Value">
</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueEmptyReturn</string>
			<bool name="Value"></bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueEmptyVTab</string>
			<bool name="Value"></bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueEmptyFormFeed</string>
			<bool name="Value"></bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueEmptyMix</string>
			<bool name="Value">
 	</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueEmptyMixCRLF</string>
			<bool name="Value">
 	</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueLeadingSpace</string>
			<bool name="Value">    true</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueLeadingTab</string>
			<bool name="Value">	true</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueLeadingNewline</string>
			<bool name="Value">
true</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueLeadingReturn</string>
			<bool name="Value">true</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueLeadingVTab</string>
			<bool name="Value">true</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueLeadingFormFeed</string>
			<bool name="Value">true</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueLeadingMix</string>
			<bool name="Value">
 	true</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueLeadingMixCRLF</string>
			<bool name="Value">
 	true</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueTrailingSpace</string>
			<bool name="Value">true    </bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueTrailingTab</string>
			<bool name="Value">true	</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueTrailingNewline</string>
			<bool name="Value">true
</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueTrailingReturn</string>
			<bool name="Value">true</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueTrailingVTab</string>
			<bool name="Value">true</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueTrailingFormFeed</string>
			<bool name="Value">true</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueTrailingMix</string>
			<bool name="Value">true
 	</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueTrailingMixCRLF</string>
			<bool name="Value">true
 	</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueWrappingSpace</string>
			<bool name="Value">    true    </bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueWrappingTab</string>
			<bool name="Value">	true	</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueWrappingNewline</string>
			<bool name="Value">
true
</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueWrappingReturn</string>
			<bool name="Value">true</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueWrappingVTab</string>
			<bool name="Value">true</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueWrappingFormFeed</string>
			<bool name="Value">true</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueWrappingMix</string>
			<bool name="Value">
 	true
 	</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueWrappingMixCRLF</string>
			<bool name="Value">
 	true
 	</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueWrappingIndent</string>
			<bool name="Value">
				true
			</bool>
		</Properties>
	</Item>
	<Item class="BoolValue">
		<Properties>
			<string name="Name">TrueWrappingIndentCRLF</string>
			<bool name="Value">
				true
			</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseBase</string>
			<bool name="Visible">false</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseEmpty</string>
			<bool name="Visible"></bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseEmptySpace</string>
			<bool name="Visible">    </bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseEmptyTab</string>
			<bool name="Visible">	</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseEmptyNewline</string>
			<bool name="Visible">
</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseEmptyReturn</string>
			<bool name="Visible"></bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseEmptyVTab</string>
			<bool name="Visible"></bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseEmptyFormFeed</string>
			<bool name="Visible"></bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseEmptyMix</string>
			<bool name="Visible">
 	</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseEmptyMixCRLF</string>
			<bool name="Visible">
 	</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseLeadingSpace</string>
			<bool name="Visible">    false</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseLeadingTab</string>
			<bool name="Visible">	false</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseLeadingNewline</string>
			<bool name="Visible">
false</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseLeadingReturn</string>
			<bool name="Visible">false</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseLeadingVTab</string>
			<bool name="Visible">false</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseLeadingFormFeed</string>
			<bool name="Visible">false</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseLeadingMix</string>
			<bool name="Visible">
 	false</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseLeadingMixCRLF</string>
			<bool name="Visible">
 	false</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseTrailingSpace</string>
			<bool name="Visible">false    </bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseTrailingTab</string>
			<bool name="Visible">false	</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseTrailingNewline</string>
			<bool name="Visible">false
</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseTrailingReturn</string>
			<bool name="Visible">false</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseTrailingVTab</string>
			<bool name="Visible">false</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseTrailingFormFeed</string>
			<bool name="Visible">false</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseTrailingMix</string>
			<bool name="Visible">false
 	</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseTrailingMixCRLF</string>
			<bool name="Visible">false
 	</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseWrappingSpace</string>
			<bool name="Visible">    false    </bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseWrappingTab</string>
			<bool name="Visible">	false	</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseWrappingNewline</string>
			<bool name="Visible">
false
</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseWrappingReturn</string>
			<bool name="Visible">false</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseWrappingVTab</string>
			<bool name="Visible">false</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseWrappingFormFeed</string>
			<bool name="Visible">false</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseWrappingMix</string>
			<bool name="Visible">
 	false
 	</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseWrappingMixCRLF</string>
			<bool name="Visible">
 	false
 	</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseWrappingIndent</string>
			<bool name="Visible">
				false
			</bool>
		</Properties>
	</Item>
	<Item class="Frame">
		<Properties>
			<string name="Name">FalseWrappingIndentCRLF</string>
			<bool name="Visible">
				false
			</bool>
		</Properties>
	</Item>
</roblox>
