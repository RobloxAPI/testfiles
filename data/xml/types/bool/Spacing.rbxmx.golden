Format: rbxmx
Output: model
Data
	Metadata
	Instances
		0
			ClassName: "BoolValue"
			IsService: false
			Reference: 0
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueBase"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		1
			ClassName: "BoolValue"
			IsService: false
			Reference: 1
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueEmpty"
			Children
		2
			ClassName: "BoolValue"
			IsService: false
			Reference: 2
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueEmptySpace"
			Children
		3
			ClassName: "BoolValue"
			IsService: false
			Reference: 3
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueEmptyTab"
			Children
		4
			ClassName: "BoolValue"
			IsService: false
			Reference: 4
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueEmptyNewline"
			Children
		5
			ClassName: "BoolValue"
			IsService: false
			Reference: 5
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueEmptyReturn"
			Children
		6
			ClassName: "BoolValue"
			IsService: false
			Reference: 6
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueEmptyVTab"
			Children
		7
			ClassName: "BoolValue"
			IsService: false
			Reference: 7
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueEmptyFormFeed"
			Children
		8
			ClassName: "BoolValue"
			IsService: false
			Reference: 8
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueEmptyMix"
			Children
		9
			ClassName: "BoolValue"
			IsService: false
			Reference: 9
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueEmptyMixCRLF"
			Children
		10
			ClassName: "BoolValue"
			IsService: false
			Reference: 10
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueLeadingSpace"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		11
			ClassName: "BoolValue"
			IsService: false
			Reference: 11
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueLeadingTab"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		12
			ClassName: "BoolValue"
			IsService: false
			Reference: 12
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueLeadingNewline"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		13
			ClassName: "BoolValue"
			IsService: false
			Reference: 13
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueLeadingReturn"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		14
			ClassName: "BoolValue"
			IsService: false
			Reference: 14
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueLeadingVTab"
			Children
		15
			ClassName: "BoolValue"
			IsService: false
			Reference: 15
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueLeadingFormFeed"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		16
			ClassName: "BoolValue"
			IsService: false
			Reference: 16
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueLeadingMix"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		17
			ClassName: "BoolValue"
			IsService: false
			Reference: 17
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueLeadingMixCRLF"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		18
			ClassName: "BoolValue"
			IsService: false
			Reference: 18
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueTrailingSpace"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		19
			ClassName: "BoolValue"
			IsService: false
			Reference: 19
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueTrailingTab"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		20
			ClassName: "BoolValue"
			IsService: false
			Reference: 20
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueTrailingNewline"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		21
			ClassName: "BoolValue"
			IsService: false
			Reference: 21
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueTrailingReturn"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		22
			ClassName: "BoolValue"
			IsService: false
			Reference: 22
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueTrailingVTab"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		23
			ClassName: "BoolValue"
			IsService: false
			Reference: 23
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueTrailingFormFeed"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		24
			ClassName: "BoolValue"
			IsService: false
			Reference: 24
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueTrailingMix"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		25
			ClassName: "BoolValue"
			IsService: false
			Reference: 25
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueTrailingMixCRLF"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		26
			ClassName: "BoolValue"
			IsService: false
			Reference: 26
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueWrappingSpace"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		27
			ClassName: "BoolValue"
			IsService: false
			Reference: 27
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueWrappingTab"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		28
			ClassName: "BoolValue"
			IsService: false
			Reference: 28
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueWrappingNewline"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		29
			ClassName: "BoolValue"
			IsService: false
			Reference: 29
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueWrappingReturn"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		30
			ClassName: "BoolValue"
			IsService: false
			Reference: 30
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueWrappingVTab"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		31
			ClassName: "BoolValue"
			IsService: false
			Reference: 31
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueWrappingFormFeed"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		32
			ClassName: "BoolValue"
			IsService: false
			Reference: 32
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueWrappingMix"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		33
			ClassName: "BoolValue"
			IsService: false
			Reference: 33
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueWrappingMixCRLF"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		34
			ClassName: "BoolValue"
			IsService: false
			Reference: 34
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueWrappingIndent"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		35
			ClassName: "BoolValue"
			IsService: false
			Reference: 35
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "TrueWrappingIndentCRLF"
				1
					Name: "Value"
					Type: "Bool"
					Value: true
			Children
		36
			ClassName: "Frame"
			IsService: false
			Reference: 36
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseBase"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		37
			ClassName: "Frame"
			IsService: false
			Reference: 37
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseEmpty"
			Children
		38
			ClassName: "Frame"
			IsService: false
			Reference: 38
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseEmptySpace"
			Children
		39
			ClassName: "Frame"
			IsService: false
			Reference: 39
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseEmptyTab"
			Children
		40
			ClassName: "Frame"
			IsService: false
			Reference: 40
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseEmptyNewline"
			Children
		41
			ClassName: "Frame"
			IsService: false
			Reference: 41
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseEmptyReturn"
			Children
		42
			ClassName: "Frame"
			IsService: false
			Reference: 42
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseEmptyVTab"
			Children
		43
			ClassName: "Frame"
			IsService: false
			Reference: 43
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseEmptyFormFeed"
			Children
		44
			ClassName: "Frame"
			IsService: false
			Reference: 44
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseEmptyMix"
			Children
		45
			ClassName: "Frame"
			IsService: false
			Reference: 45
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseEmptyMixCRLF"
			Children
		46
			ClassName: "Frame"
			IsService: false
			Reference: 46
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseLeadingSpace"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		47
			ClassName: "Frame"
			IsService: false
			Reference: 47
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseLeadingTab"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		48
			ClassName: "Frame"
			IsService: false
			Reference: 48
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseLeadingNewline"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		49
			ClassName: "Frame"
			IsService: false
			Reference: 49
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseLeadingReturn"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		50
			ClassName: "Frame"
			IsService: false
			Reference: 50
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseLeadingVTab"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		51
			ClassName: "Frame"
			IsService: false
			Reference: 51
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseLeadingFormFeed"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		52
			ClassName: "Frame"
			IsService: false
			Reference: 52
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseLeadingMix"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		53
			ClassName: "Frame"
			IsService: false
			Reference: 53
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseLeadingMixCRLF"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		54
			ClassName: "Frame"
			IsService: false
			Reference: 54
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseTrailingSpace"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		55
			ClassName: "Frame"
			IsService: false
			Reference: 55
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseTrailingTab"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		56
			ClassName: "Frame"
			IsService: false
			Reference: 56
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseTrailingNewline"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		57
			ClassName: "Frame"
			IsService: false
			Reference: 57
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseTrailingReturn"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		58
			ClassName: "Frame"
			IsService: false
			Reference: 58
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseTrailingVTab"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		59
			ClassName: "Frame"
			IsService: false
			Reference: 59
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseTrailingFormFeed"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		60
			ClassName: "Frame"
			IsService: false
			Reference: 60
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseTrailingMix"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		61
			ClassName: "Frame"
			IsService: false
			Reference: 61
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseTrailingMixCRLF"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		62
			ClassName: "Frame"
			IsService: false
			Reference: 62
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseWrappingSpace"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		63
			ClassName: "Frame"
			IsService: false
			Reference: 63
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseWrappingTab"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		64
			ClassName: "Frame"
			IsService: false
			Reference: 64
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseWrappingNewline"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		65
			ClassName: "Frame"
			IsService: false
			Reference: 65
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseWrappingReturn"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		66
			ClassName: "Frame"
			IsService: false
			Reference: 66
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseWrappingVTab"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		67
			ClassName: "Frame"
			IsService: false
			Reference: 67
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseWrappingFormFeed"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		68
			ClassName: "Frame"
			IsService: false
			Reference: 68
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseWrappingMix"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		69
			ClassName: "Frame"
			IsService: false
			Reference: 69
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseWrappingMixCRLF"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		70
			ClassName: "Frame"
			IsService: false
			Reference: 70
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseWrappingIndent"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
		71
			ClassName: "Frame"
			IsService: false
			Reference: 71
			Properties
				0
					Name: "Name"
					Type: "String"
					Value: "FalseWrappingIndentCRLF"
				1
					Name: "Visible"
					Type: "Bool"
					Value: false
			Children
